indexbuilder.go

```diff
@@ -286,6 +286,11 @@ type Document struct {
 	// Document sections for symbols. Offsets should use bytes.
 	Symbols         []DocumentSection
 	SymbolsMetaData []*Symbol
+
+	// Scores stores the score vector of a document as provided by a
+	// DocumentScoresFile file in the git repo. This field is experimental and may
+	// change at any time without warning.
+	Scores []float64
```

Comment:
I think this field should document the expectation on what scores are and what means something is good vs bad. Right now it is just documented in the sourcegraphFake implementation.